### ISSUE TITLE
Is this repo maintained?

### ISSUE THREAD

**Issue by chrisjohnson00** [MarkusMcNugen/docker-qBittorrentvpn#101]
Commit history shows only README.md updates since 2018.

There's been lots of changes implemented by users, myself included, but no PR merges from @MarkusMcNugen 

Any suggestions on moving forward with a location where collaboration occurs?
In my fork I've added ARM support.
https://github.com/MarkusMcNugen/docker-qBittorrentvpn/pull/96 adds automatic restart on VPN failure.
Both of these were hot topics and it would be great to get them out to everyone, from a central location.

I am not interested in being the sole maintainer, but would be willing to contribute my github action changes (ARM support)  and help in other ways.

**Comment by Altycoder** [comment]
Well, the oldest pull request is Jan 2019 and the oldest unresolved issue is Oct 2018...

Over the weekend I discovered gluetun by chance (https://github.com/qdm12/gluetun) so I may give that a go and then set up a torrent client only container rather than having them bundled together.

I still have issues with this container not re-establishing openvpn connections if they fail after 2-3 days of running.  It's been an open issue for a while now.

**Comment by chrisjohnson00** [comment]
> Well, the oldest pull request is Jan 2019 and the oldest unresolved issue is Oct 2018...
> 
> Over the weekend I discovered gluten by chance (https://github.com/qdm12/gluetun) so I may give that a go and then set up a torrent client only container rather than having them bundled together.
> 
> I still have issues with this container not re-establishing openvpn connections if they fail after 2-3 days of running. It's been an open issue for a while now.

Depending on how you are running your container, the work done in #96 seems to be helping some people.  Unfortunately it's not been too helpful for me, running in Kubernetes, but does make me feel more secure... knowing that it isn't reconnecting w/o VPN.

**Comment by Altycoder** [comment]
Well I'm now using gluetun and qbittorrent from linuxserver and all seems to be working fine.

The additional advantage being that I can also route other containers over gluetun as well

**Comment by chrisjohnson00** [comment]
> Well I'm now using gluetun and qbittorrent from linuxserver and all seems to be working fine.
> 
> The additional advantage being that I can also route other containers over gluetun as well

Sounds cool, I'm going to look into it later!

**Comment by 1RandomDev** [comment]
> Well I'm now using gluetun and qbittorrent from linuxserver and all seems to be working fine.
> 
> The additional advantage being that I can also route other containers over gluetun as well

Is this combination 100% secure so that there's never sent any request with my real IP? What if the VPN or gluetun fails?

**Comment by Altycoder** [comment]
Yes, I've set up qbittorrent to only use tun0 interface so if gluetun goes down, then qbittorrent has no network connection

**Comment by chrisjohnson00** [comment]
Seems this isn't maintained... and I've moved on from using this as well. Following the suggestions provided here I'm now using a Docker Compose deployment with Gluten and qbittorrent with great success.
